### ISSUE TITLE
Columns reflow patch has a missing part

### ISSUE THREAD

**Issue by gokberkgunes** [BeyondMagic/st#2]
Hello, below part is missing in the columns rows reflow patch. When this part is missing, scrolling through the terminal is not working. 

```
diff --git a/st.c b/st.c
void
@@ -3102,7 +3241,7 @@ drawregion(int x1, int y1, int x2, int y2)
 			continue;

 		term.dirty[y] = 0;
-		xdrawline(term.line[y], x1, y, x2);
+		xdrawline(TLINE(y), x1, y, x2);
 	}
 }
```

**Comment by BeyondMagic** [comment]
I will fix it soon.

**Comment by BeyondMagic** [comment]
Fixed on: 5b1aa24d813a6cc5bca499b8fdcfb78faa5e36fd.
